extensions/wikia/TemplateClassification/styles/TemplateClassification.scss

```diff
@@ -128,3 +128,8 @@ $tc-name-color-checked: $color-links;
 		text-align: right;
 	}
 }
+
+// for new templates in order to hide editor while TCS modal is visible
+.show-template-classification-modal .WikiaPage {
+	opacity: 0;
+}
```

Comment:
do we wanna go with kind of `fade-in` or other sweet UX stuff when revealing source editor? ^^